core/src/main/java/io/neow3j/protocol/http/HttpService.java

```diff
@@ -255,15 +255,14 @@ private static OkHttpClient createOkHttpClient() {
     private static void configureLogging(OkHttpClient.Builder builder) {
         if (log.isDebugEnabled()) {
             HttpLoggingInterceptor logging = new HttpLoggingInterceptor(log::debug);
-            logging.setLevel(HttpLoggingInterceptor.Level.BODY);
+            logging.level(HttpLoggingInterceptor.Level.BODY);
             builder.addInterceptor(logging);
         }
     }
 
     @Override
     protected InputStream performIO(String request) throws IOException {
-
-        RequestBody requestBody = RequestBody.create(JSON_MEDIA_TYPE, request);
+        RequestBody requestBody = RequestBody.create(request, JSON_MEDIA_TYPE);
```

Comment:
Seriously? They changed the order of params keeping the same method name?! 🤯

Reply:
Yes, that's actually the case :laughing: